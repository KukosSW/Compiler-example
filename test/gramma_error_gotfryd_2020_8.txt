DECLARE
  a, b(11:10)
BEGIN
  b(10) := 1;
  a := b(11);
END
