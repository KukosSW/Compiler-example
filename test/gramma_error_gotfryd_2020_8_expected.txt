BLAD: Zly zakres tablicy b w linii 2
